components/dashboard/src/projects/create-project-modal/CreateProjectModal.tsx

```diff
@@ -64,7 +64,7 @@ export const CreateProjectModal: FC<Props> = ({ onClose, onCreated }) => {
                                     selectedContextURL={selectedRepo?.url}
                                     selectedConfigurationId={selectedRepo?.configurationId}
                                     onChange={setSelectedRepo}
-                                    excludeProjects
+                                    excludeConfigurations
```

Comment:
[nit] keep old names can help review (xd)

we have so many places the use configuration (in papi, and it's unnecessary to change it, but can be confused sometimes), but it actually named repository now right?

Reply:
Fair about making less changes and not change these too much 🙏 - this component did not have too many references though and we mix `.configurationId` and then things like `onlyProjects`, so I wanted to make that less confusing.

> but it actually named repository now right?

Not really, it's going to be named configurations in the code in the foreseeable future because it helps to distinguish them from actual git source repositories.